go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/miekg/dns v1.1.42
 	github.com/mitchellh/go-wordwrap v1.0.1
 	github.com/montanaflynn/stats v0.6.6
+	github.com/ooni/oohttp v0.0.0-20210817213129-35949dcf526f // indirect
```

Comment:
Why is this indirect? Do you mind running `go mod tidy` and see whether it fixes this line?